hax/hax/util.py

```diff
@@ -635,6 +696,10 @@ def ensure_motr_all_started(self, event: Event):
                 return
             wait_for_event(event, 5)
 
+    def fid2objT(self, fid: str) -> str:
```

Comment:
What strings can this method return? If we know the full vocabulary of possible return values, does it make sense to return a enum or something? That would make the contract clear.

Reply:
ObjT for a given fid, added a docstring.

Reply:
My point was that raw strings as result type is error prone and tools like mypy will fail to help if I mistype the string when comparing the result value to something.

But OK, we can address this later in another PR. Would you please add FIXME for the future?

Reply:
No, I agree. I see, this is not needed actually. Have updated the patch.